src/navigation/Navigator.js

```diff
@@ -2,7 +2,81 @@
  * mSupply Mobile
  * Sustainable Solutions (NZ) Ltd. 2020
  */
+import React from 'react';
 
 import { createStackNavigator } from '@react-navigation/stack';
+import { FinaliseButton, SyncState, BackButton, MsupplyMan } from '../widgets';
+import { navigationStyles } from '../globalStyles/navigationStyles';
+import { ROUTES, FINALISABLE_PAGES } from './constants';
+import {
+  MenuPage,
+  RealmExplorer,
+  CustomerRequisitionPage,
+  CustomerRequisitionsPage,
+  SupplierRequisitionsPage,
+  CustomerInvoicesPage,
+  SupplierInvoicePage,
+  SupplierInvoicesPage,
+  StocktakesPage,
+  StocktakeManagePage,
+  StocktakeEditPage,
+  CustomerInvoicePage,
+  SupplierRequisitionPage,
+  DispensingPage,
+  PrescriptionPage,
+  CashRegisterPage,
+  SettingsPage,
+  DashboardPage,
+  StockPage,
+} from '../pages';
+
+export const DEFAULT_SCREEN_OPTIONS = {
+  headerLeft: () => <BackButton />,
+  headerTitleAlign: 'center',
+  headerTitle: MsupplyMan,
+  headerRight: () => <SyncState />,
+  headerStyle: navigationStyles.headerStyle,
+  headerLeftContainerStyle: navigationStyles.headerLeftContainerStyle,
+  headerRightContainerStyle: navigationStyles.headerRightContainerStyle,
+};
```

Comment:
these are the default options for a screen - show the back button, msupply man and sync state.